static/js/recent_topics_ui.js

```diff
@@ -663,9 +675,8 @@ export function hide() {
     // before it completely re-rerenders.
     message_view_header.render_title_area();
 
-    // Fixes misaligned message_view and hidden
-    // floating_recipient_bar.
-    navbar_alerts.resize_app();
+    // Fire our custom event
+    $("#message_feed_container").trigger("message_feed_shown");
```

Comment:
Does this need to be here, or could we just be doing a handler on `message_feed_container` being shown?  Perhaps this would end up living in the super weirdly name `gear_menu.js` library code around that if so.